client/assets/js/draw.class.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+import Snake from './snake.class';
```

Comment:
Do you mean I only need this `import Snake from './snake';` ? Because when I try to load it this way, it doesn't find it.